### PR TITLE
chore: service-contracts auto-pr with updated code snippets [DEVX-168]

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -106,10 +106,18 @@ podTemplate(
             println("This version: " + currentVersion)
             def versionExists = sh(returnStdout: true, script: 'pipenv run python3 cognite/client/utils/_version_checker.py -p cognite-sdk -v ' + currentVersion).trim()
             println("Version Exists: " + versionExists)
+
             if (env.BRANCH_NAME == 'master' && versionExists == 'no') {
                 stage('Release') {
                     sh("pipenv run twine upload --config-file /pypi/.pypirc dist/*")
                 }
+                stage('After release') {
```
Comment (sanderland) [inline Jenkinsfile]: This is the name that shows up in jenkins right, can we name it something sensible like 'Update code snippets'

Comment (polomani) [reply Jenkinsfile]: done

File: generate_code_snippets.py [hunk 2]
```diff
@@ -51,4 +51,7 @@ def collect_apis(obj, done):
             if openapi_ident in duplicate_operations:
                 snippets["operations"][duplicate_operations[openapi_ident]] += code
 
-print(json.dumps(snippets, indent=4))
+filename = "python-sdk-examples.json"
```
Comment (fredrik955) [inline generate_code_snippets.py]: I would recommend to run the script with `python3 generate_code_snippets.py > python-sdk-examples.json` instead. More flexible and less code

Comment (polomani) [reply generate_code_snippets.py]: fixed

File: Jenkinsfile [hunk 3]
```diff
@@ -106,10 +106,18 @@ podTemplate(
             println("This version: " + currentVersion)
             def versionExists = sh(returnStdout: true, script: 'pipenv run python3 cognite/client/utils/_version_checker.py -p cognite-sdk -v ' + currentVersion).trim()
             println("Version Exists: " + versionExists)
+
             if (env.BRANCH_NAME == 'master' && versionExists == 'no') {
                 stage('Release') {
                     sh("pipenv run twine upload --config-file /pypi/.pypirc dist/*")
                 }
+                stage('Update code snippets on service-contracts') {
+                    sh("pip3 install requests")
```
Comment (erlendvollset) [inline Jenkinsfile]: Requests should already be installed.

File: Jenkinsfile [hunk 4]
```diff
@@ -106,10 +106,18 @@ podTemplate(
...
+                    sh("python3 generate_code_snippets.py > python-sdk-examples.json")
```
Comment (erlendvollset) [inline Jenkinsfile]:
```suggestion
                    sh("python3 scripts/generate_code_snippets.py > python-sdk-examples.json")
```
?

File: Jenkinsfile (hunk 3 above)
Comment (polomani) [reply Jenkinsfile]: you sure? it did fail for me last time though

Comment (polomani) [reply Jenkinsfile]: ok, we realised that I need `pipenv`

**Pull request merged**